packages/vapor/scss/components/badge.scss

```diff
@@ -1,21 +1,69 @@
 .badge {
+    // Default Style
     display: inline-block;
     box-sizing: border-box;
-    height: var(--badge-height);
-    margin-right: var(--badge-margin);
-    padding: var(--badge-padding);
-    color: var(--badge-text-color);
-    font-weight: var(--badge-font-weight);
-    line-height: var(--badge-line-height);
+
+    // Style override
+    height: var(--height);
+    padding: var(--padding);
+    color: var(--color);
+    font-weight: var(--main-font-bolder);
+    font-size: var(--font-size);
+    line-height: var(--line-height);
     text-align: center;
-    background-color: var(--badge-background-color);
-    border: var(--badge-border);
+    text-transform: capitalize;
+    background-color: var(--background-color);
+    border: var(--default-border);
+    border-color: var(--border-color);
+    border-radius: 8px;
```

Comment:
```suggestion
    border-radius: var(--border-radius);
```

Reply:
Not sure about this, but maybe `mod-small` badge would have a smaller border-radius?

Reply:
To make it more square? :thinking:

Reply:
The idea is to create only variable if we override him. In this case, I always use border-radius 8px so I dont create a variable for that.